Sources/ssg/template.swift

```diff
@@ -12,67 +12,57 @@ enum Template {
 static let keyFile: String =
 """
 // ****
-// this file was automatically generated. do not edit
+// This file was automatically generated; do not edit.
 //
-// 
-import Foundation
-import CryptoKit
-
-enum ProjectKeys {
 
-    private static var keyData: [UInt8] { {{encryptionKey}} }
-
-    private static var decryptKey: SymmetricKey? {
-        guard let d = dataFromBase64Encoded(keyData) else { return nil }
-        return SymmetricKey(data: d)
-    }
+import CryptoKit
 
+public enum ProjectKeys: CaseIterable {
     {{#apiKeys}}
-    private static var {{keyName}}value: [UInt8] { {{keyValue}} }
+    case {{keyName}}
     {{/apiKeys}}
 
-    private enum Internal {
+    public var value: String { reconstitute }
+}
+
+private extension ProjectKeys {
+    var bytes: [UInt8] {
+        switch self {
         {{#apiKeys}}
-        case {{keyName}}
+        case .{{keyName}}: return Self.{{keyName}}ScrabbledValue
```

Comment:
I think you might have meant "Scrambled" instead of "Scrabbled".

Reply:
I pushed a fix for the typo.